src/resources.rs

```diff
@@ -148,7 +153,7 @@ fn inspect_repr(repr: &Repr) -> String {
 
 // Abstract async file interface.
 // Ideally in unix, if Resource represents an OS rid, it will be the same.
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Default)]
```

Comment:
Does Default make sense here?

Reply:
Hmm not really. It was added originally with the experiment of adding more fields to `Resource` struct, but that's no longer the case